test/e2e-common.sh

```diff
@@ -32,9 +32,15 @@ function add_sidecar_registry() {
     rm -f ${TMPF}.read
 }
 
-# Add the git_clone task
-function add_git_clone_task() {
-    kubectl -n ${tns} apply -f ./task/git-clone/0.1/git-clone.yaml
+function add_task() {
+    local array version task
+    task=${1}
+    version=${2}
+    [[ "${version}" == "latest" ]] && {
+        array=($(echo task/${task}/*|sort -u))
+        version=${array[-1]}
```

Comment:
nit: can we have a better name for this, as it will be a path like `task/git-clone/0.2` it avoids confusion in the next apply step

Reply:
better name of what ? the name of the function?

Reply:
name of the var `version`, it confused me, maybe it's only me :smile: 

Because in the next step we do `kubectl -n "${tns}" apply -f "${version}"/"${task}".yaml`

Reply:
ah yeah you want to be named path_version or something... sure i'll be happy to rename so you are not confused if you reread this code :-p

Reply:
there is actually a bug... :)